modules/update-manifest-application.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="update-manifest-application_{context}"]
+= Understanding how manifests are applied during an update
+
+Some manifests supplied in a release image need to be applied in a certain order because of the dependencies between them.
```

Comment:
>the CVO applies must apply 

The CVO applies

@dfitzmau  right?

Reply:
Per the IBM style guide:

"Passive voice is acceptable when any of these conditions are true:

- The system performs the action.
- It is more appropriate to focus on the receiver of the action.
- You want to avoid blaming the user for an error, such as in an error message.
- The information is clearer in passive voice."

I think the first (and even the second) bullet point applies here, and I think it might read better if the lengthy noun phrase "manifests supplied in a release image" is towards the beginning of the sentence rather than the middle.

For those reasons I'd prefer to keep it in passive, but please let me know if you disagree. 
I will, however, replace "need to" with "must" as that's the preferred word per the style guide.